lib/ts/recipe/session/recipe.ts

```diff
@@ -197,7 +197,7 @@ export default class SessionRecipe extends RecipeModule {
                     err.payload.clearTokens === true
```

Comment:
in the index.ts file of this recipe, we want to change all the functions that end with `WithoutRequestResponse` to also take an optional antiCsrf option whose by default value will be undefined.

If the user's origin is a string and user has passed undefined, we can then calculate the right value of the anti csrf by calling the function with an empty request object., else we throw an error.